R/helper_functions.R

```diff
@@ -1,23 +1,18 @@
 # Helper functions
-save_qs <- function(qsObj, qsName) {
-  qs::qsave(
-    qsObj,
-    file.path(system.file("testdata", package = "gDRtestData"), qsName), 
-    compress = "gzip"
-  )
-}
-
+#' @keywords internal
 prepareData <- function(cell_lines, drugs, conc = 10 ^ (seq(-3, 1, 0.5))) {
-  df_layout <- data.table::setDT(merge.data.frame(cell_lines, drugs, by = NULL))
+  df_layout <- drugs[, as.list(cell_lines), names(drugs)]
   df_layout <- add_data_replicates(df_layout)
   add_concentration(df_layout, conc)
 }
 
+#' @keywords internal
```

Comment:
If you do not export this function you may not use them in gDRcore like `gDRtestData::prepareMergedData` :(

Reply:
done